_posts/2024-06-18-trino-fest-recap.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "Trino Fest 2024 recap"
+author: "Manfred Moser, Cole Bowden, Monica Miller"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2024/recap-banner.png
+---
+
+tbd
+Trino Fest 2024 is successfully in the books! While over 100 enthusiastic
+members of the community gathered in Boston, over 650 virtual attendees
+joined us worldwide to learn from our expert speakers as they discussed
+topics such as table formats, enhancements and optimizations, and use
+cases with Trino both large and small.
+<!--more-->
+
+## Re
+tbd
+
+
+## Sessions
+
+* [What’s new in Trino this summer](https://www.youtube.com/watch?v=mk3n0_tAdZY){:target="_blank"}
+  presented by Martin Traverso from
+  [Starburst](https://www.starburst.io){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Reducing query cost and query runtimes of Trino powered analytics platforms](https://www.youtube.com/watch?v=11yhPXIXiBY){:target="_blank"}
+  presented by Jonas Irgens Kylling from
+  [Dune](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Enhancing Trino's query performance and data management with Hudi: innovations and future](https://www.youtube.com/watch?v=JMzS2BbeK0E){:target="_blank"}
+  presented by Ethan Guo from
+  [Onehouse](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Trino Engineering @ Microsoft](https://www.youtube.com/watch?v=t7ndqYUhKSA){:target="_blank"}
+  presented by George Fisher and Ishan Patwa from
+  [Microsoft](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Enhancing data governance in Trino with the OpenLineage integration](https://www.youtube.com/watch?v=A7hj1M7IYj8){:target="_blank"}
+  presented by Alok Kumar Prusty from
+  [Apple](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Best practices and insights when migrating to Apache Iceberg for data engineers](https://www.youtube.com/watch?v=dKQ2zShNlyQ){:target="_blank"}
```

Comment:
Amit shared how Cloudinary expanded their data lake to use Apache Iceberg. He demonstrated how moving from Snowflake to an open table format allowed them to reduce storage costs and leverage different query and processing engines to run more powerful analytics at scale.